src/atwho.js

```diff
@@ -109,7 +109,7 @@ define([
         });
 
         if (options.useRichText) {
-            options.insertTpl = '${name}';
+            options.insertTpl = '${hashtagPath}';
```

Comment:
This was interesting. `fuse.js` returns `name` as the absolute path (`/data/....`) so when using rich text it was insterting /data/ and the rich text code was converting that to hashtags.